docs/src/user-guide/command-line-interface.md

```diff
@@ -370,83 +390,86 @@ npx eslint --quiet file.js
 
 This option allows you to specify a warning threshold, which can be used to force ESLint to exit with an error status if there are too many warning-level rule violations in your project.
 
-Normally, if ESLint runs and finds no errors (only warnings), it exits with a success exit status. However, if `--max-warnings` is specified and the total warning count is greater than the specified threshold, ESLint exits with an error status. To prevent this behavior, omit this option or specify a threshold of `-1`.
+* **Argument Type**: Integer. Maximum number of warnings to allow. To prevent this behavior, do not use this option or specify `-1` as the argument.
+* **Multiple Arguments**: No
 
-Example:
+Normally, if ESLint runs and finds no errors (only warnings), it exits with a success exit status. However, if `--max-warnings` is specified and the total warning count is greater than the specified threshold, ESLint exits with an error status.
+
+##### `--max-warnings` example
 
 ```shell
 npx eslint --max-warnings 10 file.js
 ```
 
 ### Output
 
-#### `-o`, `--output-file`
+#### `--output-file`
+
+Write output of linting results to a specified file.
 
-Enable report to be written to a file.
+* **Alias**: `-o`
+* **Argument Type**: String. Path to file.
+* **Multiple Arguments**: No
 
-Example:
+##### `--output-file` example
 
 ```shell
 npx eslint -o ./test/test.html
 ```
 
-When specified, the given format is output into the provided file name.
+#### `--format`
 
-#### `-f`, `--format`
+This option specifies the output format for the console.
 
-This option specifies the output format for the console. Possible formats are:
+* **Alias**: `-f`
+* **Argument Type**: String. One of the [built-in formatters](formatters) or a custom formatter.
```

Comment:
```suggestion
* **Argument Type**: String. One of the [built-in formatters](formatters/) or a custom formatter.
```

Reply:
the link as it is seems to work fine. any reason to update this?

Reply:
So that internal links always point to canonical URLs. For the formatters page, it's: https://eslint.org/docs/latest/user-guide/formatters/.

https://eslint.org/docs/latest/user-guide/formatters should actually redirect to https://eslint.org/docs/latest/user-guide/formatters/, but that seems impossible to configure on Netlify when proxying the content. We've managed to make the pages work mostly well when opened through non-canonical URLs, but it's still preferable to direct users to canonical ones.